webex/react-widgets
# Unregister device when removing widget

This PR handle the issue mentioned on this ticket: Ticket: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-147741

Basically when we "Remove Widget" we also make sure that we remove (DELETE) the device.

Steps to test it: 
1) Add you Access Token
2) In the "Space Widget" section click on "Open Space Widget"
3) Open the Developer console and go to the "Network" tab 
4) Click on "Remove Widget"
5) You should see a DELETE call to `https://wdm-a.wbx2.com/wdm/api/v1/devices/<<devideId>>`

![Screen Shot 2020-06-23 at 12 09 18 PM](https://user-images.githubusercontent.com/3578942/85428174-d260fb80-b54a-11ea-8ecf-75a35747400c.png)
